src/Form/PublisherFormType.php

```diff
@@ -4,19 +4,40 @@
 
 use App\Entity\Publisher;
 use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
 use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Constraints\NotBlank;
 
 class PublisherFormType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('name')
-            ->add('value')
-            ->add('country')
-            ->add('year')
+            ->add('name', TextType::class, [
```

Comment:
siaip visad uzdek constraintus ilgio
, nes dazniausiai ant stringu DB yra limitas (gali paziureti per koki DB gui) tai atveri savo koda luzimams, ir overall, jeigu dabar kazkas suves pavadinimas is kokiu 400 raidziu, klausimas kaip tavo front'as atrodys su tokiais stringais :D